app/models/child.rb

```diff
@@ -127,7 +127,7 @@ def initialize(attributes = {})
   end
 
```

Comment:
Ici tu peux ajouter par exemple :
```
def family_tag_list=(values)
  family.tag_list = values
end
```

Reply:
Peut-être même que ton `delegate :tag_list, to: :family, prefix: true` fait déjà ça pour toi